docs/docs/build/how-to/ordinals.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 5
+---
+
+# Ordinals
+
+Ordinals are a system for tracking and transferring satoshis, Bitcoin's smallest units and attaching data to satoshis. When such attachment of data to satoshi happens an __inscription__ is created. Inscription content is entirely on-chain, stored in taproot script-path spend scripts forever. These scripts receive witness discounts, making inscription content storage relatively economical.
+
+
+## How to create an inscription
+Inscriptions are created using taproot script. That means you have to use a P2TR type of address to be able to create the inscription. 
+
+### Wallets that support taproot addresses
+
+- [Sparrow wallet](https://sparrowwallet.com/) (Desktop)
+- [Xverse](https://www.xverse.app/download) (Browser and mobile)
+- [Unisat](https://unisat.io/download ) (Browser)
```

Comment:
Would also add the ord wallet https://docs.ordinals.com/guides/inscriptions.html